documentation/aircraft-commands.md

```diff
@@ -164,28 +161,36 @@ These commands allow you to manipulate the route in the aircraft's FMS.
 ~~_Aliases -_ `f` / `fix` / `track`~~
 
 ~~_Syntax -_ `AAL123 f [fixname]`~~
+
 *_This command has been deprecated_*
 
 ### Hold
 
 _Aliases -_ `hold`
 
 _Information -_ This command instructs the aircraft to enter a holding
-pattern until further notice. The direction (left/right) may be specified,
-as well as the leg length (in minutes), as well as the fix to hold over.
-But you may also omit those specifications, in which case, the aircraft
-will enter a standard holding pattern over their present position (right
-turns, 1 minute legs, using the reciprocal bearing to the fix as the outbound
-radial). To clear the aircraft out of the hold, either clear it direct to
-a fix or assign it a new heading.
-
-_Parameters -_ Specify the direction of turns during the hold with `right`
-or `left`, the leg length, with `[time]min`, the fix to hold over
-with simply `[fixname]` and the `radial` (a 3-digit course, eg. 001 to 360) which
-defines the outbound leg. Any combination of these arguments provided in any
-order is acceptable, as long as the command `hold` comes first.
-
-_Syntax -_ `AAL123 hold [fixname] [left|right] [leg_time]min [radial]` or `AAL123 hold`
+pattern over a specified fix until further notice. (If no fix is provided, it
+is intended that the aircraft should hold over their present position, but this
+is currently not implemented.)
+
+By default, the aircraft will fly a standard holding pattern, i.e. right turns,
+1 minute legs, and using the reciprocal bearing to the fix as the outbound
+radial. You may override this by specifying any of: the direction of the hold
+(left/right), the leg length (either in minutes or nautical miles), and/or the
+heading for the outbound radial.
+
+_Parameters -_
+Specify the the fix to hold over with simply `[fixname]`.  
+Optionally, you may also specify:
+
+- the direction of turns during the hold with `left`, `l`, `right`, or `r`
+- the leg length, either as `[time]min` or `[distance]nm`
+- the `radial` (a 3-digit course, eg. 001 to 360) which defines the outbound leg.
+
+Any combination of these arguments provided in any order is acceptable, as long
+as the command `hold` comes first.
+
+_Syntax -_ `AAL123 hold [fixname] [left|l|right|r] [leg_time]min|[leg_dist]nm [radial]` ~~or `AAL123 hold`~~
```

Comment:
Please add this sentence back-- I know it is not specific to this command, but is important since the `exitHold` command is not the only way to do so, and may not even be working.

>To clear the aircraft out of the hold, you can use the `exitHold` command below, clear it direct to a fix, or assign it a new heading.

Reply:
done